waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManager.java

```diff
@@ -116,7 +116,7 @@ void open() {
 
     for (int attempt = 0; !isConnected && (attempt < retries); ++attempt) {
       for (URI store : metastoreUris) {
-        LOG.info("Trying to connect to metastore with URI " + store);
+        LOG.debug("Trying to connect to metastore with URI " + store);
```

Comment:
Having these messages at `info` level is pretty helpful, especially while we are trying to debug issues.  And also since line 172/175 doesn't log which metastore failed to connect, so you need the above message to match it up.

Reply:
No problem put it back changed so other events also print the URIs so we don't have to piece those things together.